src/Implementation/ColdStorage/ColdStorage/BufferedFrameData.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Practices.IoTJourney.ColdStorage.RollingBlobWriter;
+using Microsoft.ServiceBus.Messaging;
+
+namespace Microsoft.Practices.IoTJourney.ColdStorage
+{
+    public class BufferedFrameData : BlockData
+    {
+        private readonly EventData _lastEventDataInFrame;
```

Comment:
Since BufferedFrameData is passed an instance of EventData and just exposes it as a property, I don't think it would be appropriate for BufferedFrameData to participate in the disposing of EventData.